spec/lib/tasks/dupicate_rent_periods_spec.rb

```diff
@@ -0,0 +1,47 @@
+require "rails_helper"
+require "rake"
+
+RSpec.describe "duplicate_rent_periods" do
+  before do
+    Rake.application.rake_require("tasks/duplicate_rent_periods")
+    Rake::Task.define_task(:environment)
+    allow(Rails.logger).to receive(:info)
+    organisation = create(:organisation, rent_periods: (1..11).to_a)
+
+    [2, 4, 6, 11].each do |rent_period|
+      org_rent_period = build(:organisation_rent_period, organisation:)
+      org_rent_period.save!(validate: false)
+      org_rent_period.update_column(:rent_period, rent_period)
```

Comment:
Could you set the rent period in the initial construction (or otherwise before the save) to avoid needing two db updates? (That will make a difference to how long the tests take to run - not huge for just these but as a general principle we want to keep them as fast as we can)

Reply:
Setting the rent_periods directly in the constructor for the organisation variable won't work because duplicates won't be saved that way, even with validate:false. However, I can set the rent_periods before saving and will change that.

The following approaches won't work:
`organisation = create(:organisation, rent_periods: (1..11).to_a.concat([2, 4, 6, 11, 2, 11]))`
or
`organisation = create(:organisation, rent_periods: [1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 2, 4, 6, 11, 2, 11])`